### PR TITLE
no more bugs

### PR DESCRIPTION
**Pull request by teddyoweh**
Fixed get started button content alignment to center and typo

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -10,6 +10,12 @@
       href="https://fonts.googleapis.com/css2?family=Outfit:wght@400;500;800;900&display=swap"
       rel="stylesheet"
     />
+    <style>
+      .button button_primary_light{
```
Comment (ernestechie) [inline dist/index.html]: Class selector is '.button_primary_light' not 'button_primary_light'. Please fix 😉😉

File: dist/index.html [hunk 2]
```diff
@@ -181,7 +187,7 @@ <h4 class="text_xlg">Don't just take our word for it.</h4>
           <div class="testimonials_main_content">
             <p class="text_r">
               <span class="text_primary">"</span>
-              My pelf story is an interestin gone. Pelf has made me realize that
+              My pelf story is an interesting one. Pelf has made me realize that
```
Comment (ernestechie) [inline dist/index.html:190]: Thanks, i never saw this 😄

**Review by ernestechie — COMMENTED** [review]
Correct typo

File: dist/index.html (hunk 2 above)
Comment (teddyoweh) [reply dist/index.html:190]: That's what pull requests are for!

**Review by teddyoweh — COMMENTED** [review]
Fixed the selector error

**Review by ernestechie — APPROVED** [review]
Reviewed. Thanks for your contribution

**Comment by ernestechie** [comment]
Thanks for your contribution

**Pull request merged**